01_standalone_examples/databricks-ci-cd/README.md

```diff
@@ -0,0 +1,222 @@
+
+## Prerequisites
+* lakeFS installed and running on a server or in the cloud. If you don't have lakeFS already running then either use [lakeFS Cloud](https://demo.lakefs.io/) which provides free lakeFS server on-demand with a single click or refer to [lakeFS Quickstart](https://docs.lakefs.io/quickstart/) doc.
+* Databricks server with the ability to run compute clusters on top of it. 
+* Configure your Databricks cluster to use lakeFS Hadoop file system. Read this blog [Databricks and lakeFS Integration: Step-by-Step Configuration Tutorial](https://lakefs.io/blog/databricks-lakefs-integration-tutorial/) or [lakeFS documentation](https://docs.lakefs.io/integrations/spark.html#lakefs-hadoop-filesystem) for the configuration.
+* Permissions to manage the cluster configuration, including adding libraries. 
+* GitHub account. 
+
+## Setup
+
+1. Create [Databricks personal access token](https://docs.databricks.com/en/dev-tools/auth/pat.html).
+
+
+1. Create Databricks secret scope e.g. **demos** or use an existing secret scope. Add following secrets in that secret scope by following [Secret management docs](https://docs.databricks.com/en/security/secrets/index.html): 
+
+        lakefs_access_key_id e.g. 'AKIAIOSFOLKFSSAMPLES'
+
+        lakefs_secret_access_key e.g. 'wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY'
+
+1. Create a Git repository. It can be named **lakeFS-samples-ci-cd**.
+
+1. Clone this repository:
+
+   ```bash
+   git clone https://github.com/treeverse/lakeFS-samples && cd lakeFS-samples/01_standalone_examples/aws-databricks/ci-cd-demo
+   ```
+
+1. Create folders **.github/workflows** and **databricks-notebooks** in your Git repo.
+
+1. Upload **pr_commit_run_databricks_etl_job.yml** file in **lakeFS-samples/01_standalone_examples/aws-databricks/ci-cd-demo/.github/workflows** folder to **.github/workflows** folder in your Git repo.
+
+1. Upload all files in **lakeFS-samples/01_standalone_examples/aws-databricks/ci-cd-demo/databricks-notebooks** folder to **databricks-notebooks** folder in your Git repo.
+
+1. Add following secrets in your Git repo by following [Creating secrets for a repository docs](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#creating-secrets-for-a-repository). This is the Databricks token created in 1st step above.
+
+        DATABRICKS_TOKEN
+
+
+1. Add following variables in your Git repo by following [Creating configuration variables for a repository docs](https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-a-repository):
+* Variable to store your [Databricks host name or URL](https://docs.databricks.com/en/workspace/workspace-details.html#workspace-instance-names-urls-and-ids) e.g. https://cust-success.cloud.databricks.com
+
+        DATABRICKS_HOST 
```

Comment:
When you copy this there are extra spaces at the begining - the copy should be only for `DATABRICKS_HOSTS`